sway/input/seat.c

```diff
@@ -1490,6 +1490,8 @@ void seat_apply_config(struct sway_seat *seat,
 	}
 
 	seat->config = seat_config;
+	seat->idle_inhibit_sources = seat_config->idle_inhibit_sources;
+	seat->idle_wake_sources = seat_config->idle_wake_sources;
```

Comment:
Not sure why I removed these to begin with.

Weirdly enough, idle inhibition still worked even with these removed, or maybe I'm not understanding what these actually do.